otx/api/usecases/exportable_code/demo/demo_package/model_container.py

```diff
@@ -28,7 +29,14 @@ class ModelContainer:
     """
 
     def __init__(self, model_dir: Path, device="CPU") -> None:
-        self.parameters = get_parameters(model_dir / "config.json")
+        model_adapter = OpenvinoAdapter(create_core(), get_model_path(model_dir / "model.xml"), device=device)
+
+        try:
+            config_data = model_adapter.model.get_rt_info(["otx_config"])
+            self.parameters = json.loads(config_data)
+        except RuntimeError:
+            self.parameters = get_parameters(model_dir / "config.json")
+
```

Comment:
Could you check if this block could guarantee the backward compatibility for models exported from OTE0.x (which is being used in Geti1.0~3)?
cc: @eugene123tw

Reply:
@goodsong81 the tests were passed, indicating that IRs without embedded config don't crash the exportable code demos.

Reply:
@goodsong81 the fix you mentioned was already applied in detection openvino task and it's not related to this PR:
https://github.com/openvinotoolkit/training_extensions/blob/develop/otx/algorithms/detection/tasks/openvino.py#L333
https://github.com/openvinotoolkit/training_extensions/commit/7dc7b8b42874b2a5df50a81a849381906cc17ee7